bundles/org.eclipse.equinox.p2.core/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.equinox.p2.core;singleton:=true
-Bundle-Version: 2.12.200.qualifier
+Bundle-Version: 2.13.100.qualifier
```

Comment:
This is almost right.  This should be 2.13.0.
```suggestion
Bundle-Version: 2.13.0.qualifier
```

Reply:
Done.

Reply:
Very good.